### ISSUE TITLE
MenuItem can't be wrapped in anchor

### ISSUE THREAD

**Issue by oste** [jamesmfriedman/rmwc#311]
## RMWC or MDC?
RMWC

### What RMWC Version are you using?

> 2.1.0

### What browser(s) and React Version is this bug affecting?

> Chrome 68 React 16.5

### What build system are you using?

> Create React App

### What are the steps to reproduce the bug?

This `<Link>` doesn't work wrapping a MenuItem
```
<Menu>
    <Link to={ itme.link } replace={item.replace} style={{ textDecoration: 'none', color: 'inherit' }}>
        <MenuItem className='profile-menu-item dropdown-menu-item'>
            <MaterialIcon icon={item.icon} />
        </MenuItem>
    </Link>
</Menu>
```

### What is the expected behavior?

href would work

### What is the actual behavior?

The click event is not propagated

**Comment by j-o-d-o** [comment]
I would advise to put the <Link/> element inside the <MenuItem/> element and use css to make it fill the whole </MenuItem>

**Comment by oste** [comment]
offending line of code https://github.com/material-components/material-components-web/blob/master/packages/mdc-menu/foundation.js#L221

**Comment by oste** [comment]
@j-o-d-o that won't work FYI. Even if you click the anchor it doesn't make it b/c of the prevent default shown in the link above. Putting the Link inside MenuItem is not good b/c MenuItem has padding.

**Comment by j-o-d-o** [comment]
Ah I see, this again. There was a similar issue in MDC with Drawers if I remember correctly. Maybe @jamesmfriedman  has some fix in rmwc, but for the time being I think we need to wait to get this fix in MDC :(

**Comment by j-o-d-o** [comment]
Though with a bit of css hacking you can get it working. Noone is preventing you from removing that padding from MenuItem:
https://codesandbox.io/s/x7p303qk3z

**Comment by jamesmfriedman** [comment]
Found a workaround, will be released in 3.0.0
